modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -129,33 +129,35 @@ image::applicationfile.png[ApplicationFile]
 
 === Runtime Version
 
-Using the drop-down menu, select the *Runtime version* to which you to deploy:
+Using the drop-down menu, select the *Runtime version* to which to deploy:
```

Comment:
to which to deploy isn't quite correct (and is hard to read?). How about "...select the Runtime version where you wish to deploy:"?